lib/get-wrapper.js

```diff
@@ -39,6 +39,23 @@ function getCodeStart(ast) {
   return codeStart;
 }
 
+function findUseStrictRange(astInnerBody, headEnd) {
```

Comment:
Might be a bit heavy if you don't want to move the `use strict`. Send `options.moveStrict` into this module, perhaps?